site/docs/v1/tech/samlv2/pagerduty.adoc

```diff
@@ -0,0 +1,85 @@
+
+== Support
+
+If you need help with this integration, you can request assistance in one of two ways.
+
+* If you have a plan which includes technical support, please https://account.fusionauth.io/account/support/[open a support ticket].
+* If you do not have a plan which includes technical support, link:/community/forum/[please create a forum post].
+
+You can review our link:/docs/v1/tech/admin-guide/technical-support[Technical Support documentation] for more details.
+
+== Integration Walkthrough
+
+=== In PagerDuty
+
+. The _Account Owner_ can configure the SSO settings by navigating to [breadcrumb]#User Icon -> Account Settings -> Single Sign-On#.
+. Select the *SAML* radio button.
+. Copy the *SAML Endpoint URL*.
+
+_________________________________________________________________________________________________
+For more details, please visit
+https://support.pagerduty.com/docs/sso[PagerDuty SSO support page]
+_________________________________________________________________________________________________
+
+=== In FusionAuth
+
+. Click the [uielement]#Applications# link in the sidebar.
+. Create a new Application by clicking the green *+* button in the top right corner of the page.
+. Enter a name, such as *PagerDuty*, for the app and then click on the [breadcrumb]#SAML# tab.
+. Set the [field]#Enabled# field to true to turn the integration on.
+. Configure SAML functionality to integrate with your PagerDuty account.
+* Set the [field]#Issuer# value to `\https://YOUR-SUBDOMAIN.pagerduty.com`, where `YOUR-SUBDOMAIN` is the value of your PagerDuty subdomain.
+* Leave the [field]#Audience# blank.
+* Paste the [uielement]#SAML Endpoint URL# you copied from PagerDuty into the [field]#Authorized redirect URLs# field. The value in this field will be blue when entered.
+. Save the application by clicking the *blue floppy disk icon* in the top right corner of the page.
+. When redirected back to the [breadcrumb]#Applications# page, view your application details by clicking the magnifier green icon.
+* In the dialog, scroll down to the [uielement]#SAML v2 Integration details# section.
+* Copy the [uielement]#Entity Id# URL and open it in a new browser tab. This will contain configuration you'll need shortly.
+** _Note: This address contains an XML file. If your browser doesn’t automatically format it, you can copy everything and use https://xmlviewer.org/[an online XML beautifier]_.
+** Copy the contents of the `X509Certificate` element from the XML file and paste them in PagerDuty.
+*** You should paste the contents enclosed between `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`, like the example below:
++
+[source,crt]
+....
+-----BEGIN CERTIFICATE-----
+<contents of X509Certificate copied from FusionAuth>
+-----END CERTIFICATE-----
+....
+.Here's a walkthrough of these steps.
+video::/assets/docs/v1/tech/samlv2/pagerduty-sso-certificate.webm[width=560,height=315]
```

Comment:
I followed [these steps](4375de49fb54a4b53236ee736a38ac33de79f1ef) to generate `.webm` and Safari-compliant `.mp4` files but I couldn't add them both (see asciidoctor/asciidoctor#1463) besides manually inserting a `<video>` HTML fragment. Any suggestions?

Reply:
Sure. We usually just use youtube. If you provide me an MP4 file and description, I can upload to youtube and provide you the URL.